Minnet App/src/components/organisms/AssetRows/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, styled } from "@mui/material";
+import React, { useState } from "react";
+import { rows } from "../../../utils/constants";
+import Header from "../../molecules/Header";
+import { TradeRow } from "../../molecules/TradeRow";
+import TabsMolecule from "../../molecules/TabsMolecule";
+const StyledBox = styled(Box)(({ theme }) => ({
+  height: "804px",
+  width: "1200px",
+  backgroundColor: theme.palette.background.default,
+  padding: theme.spacing(2),
+}));
+
+const AssetRows = () => {
+  const [watchedItems, setWatchedItems] = useState<Set<string>>(new Set());
+  const [tabValue, setTabValue] = useState(0);
+
+  const handleTabChange = (event: React.SyntheticEvent, newValue: number) => {
+    setTabValue(newValue);
+  };
+
+  const toggleWatch = (name: string) => {
+    setWatchedItems((prev) => {
+      const newSet = new Set(prev);
+      if (newSet.has(name)) {
+        newSet.delete(name);
+      } else {
+        newSet.add(name);
+      }
+      return newSet;
+    });
+  };
+
+  const displayRows =
+    tabValue === 1 ? rows.filter((item) => watchedItems.has(item.name)) : rows;
+
+  return (
+    <StyledBox>
+      <TabsMolecule value={tabValue} onChange={handleTabChange} />
+      <Header />
+      {displayRows.map((item, index) => (
+        <TradeRow
+          key={index}
```

Comment:
Avoid using index , instead add id in the data and use it here for key

Reply:
added id in data and using as key, resolved.